src/unk_02097720.c

```diff
@@ -0,0 +1,40 @@
+#include "unk_02097720.h"
+#include "unk_0203E348.h"
+
+void sub_02097720(TaskManager *taskManager, u16* unknownPtr) {
+    FieldSystem *fsys = TaskManager_GetSys(taskManager);
+    Unknown_02097720 *data = AllocFromHeapAtEnd(0xb, sizeof(Unknown_02097720));
```

Comment:
```suggestion
    Unknown_02097720 *data = AllocFromHeapAtEnd(11, sizeof(Unknown_02097720));
```

If it's a small number, represent it as decimal.
If there's a defined enum or constant for this value which fits the context, use that instead.
If it's a flag array and the bitmasks are named, use those bitmasks separated by the bitwise or operator (`|`).